samples/graph-meeting-notification/nodejs/.env

```diff
@@ -1,7 +1,7 @@
 MicrosoftAppId=<<MicrosoftAppId>>
 MicrosoftAppPassword=<<MicrosoftAppPassword>>
 MicrosoftAppTenantId=<<MicrosoftAppTenantId>>
-notificationUrl=<<BaseUrl>>/api/notifications
+notificationUrl=
```

Comment:
What should be the value for it?
Kindly mention in the Readme file.

Reply:
Updated Readme, please review it once.